src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,19 @@
-export const TodoList = () => {};
+import React from 'react';
+import { Todo } from '../../types';
+import { TodoInfo } from '../TodoInfo';
+
+interface Props {
+  todos: Todo[];
+}
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  const maxId = Math.max(...todos.map(todo => todo.id), 0);
+
+  return (
+    <section className="TodoList">
+      {todos.map(todo => (
+        <TodoInfo key={todo.id} todo={{ ...todo, id: maxId + 1 }} />
```

Comment:
The `id` of each `todo` is being overridden with `maxId + 1`, which is incorrect. This results in all todos having the same `id`. You should pass the original `todo.id` instead of modifying it.